### DISCUSSION TITLE
VB6 Projects

### DISCUSSION THREAD

**Discussion by mika76** [twinbasic/twinbasic#134] [Q&A] [answered]
This project sounds great, but I don't really see any specifics on how it should work with VB6 projects?

**Comment by bclothier** [comment, marked as answer]
It sounds like you expect it to work with `.vbp` project. This is not yet natively supported in the preview as listed in the limitation. 

[From website](https://twinbasic.com/preview.html):
> PREVIEW NOTE:    In a future revision we will offer automatic migration from VB6 project files (.vbp). We will also allow the project file to hold links to external files, as the files are currently stored directly into the twinproj filesystem. There will also be support for VB6 .BAS and .CLS files in the future versions.

For now, you can copy'n'paste the VB6 code from your project to tB project for testing out the preview.

**Reply by mika76** [reply]
Ah thanks, I missed that part. Is there maybe a github issue I could subscribe to to see when this feature becomes available?
